common/wallet_tx.c

```diff
@@ -39,6 +46,37 @@ struct command_result *param_wtx(struct command *cmd,
         return NULL;
 }
 
+struct command_result *param_wtx_utxos(struct command *cmd,
+				 const char *name,
+				 const char *buffer,
+				 const jsmntok_t *tok,
+				 struct wallet_tx *wtx)
+{
+	size_t i;
+	const jsmntok_t *curr;
+	const char **outpoints = tal_arr(cmd, const char*, 0);
+
+	json_for_each_arr(i, curr, tok)
+		tal_arr_expand(&outpoints, json_strdup(tmpctx, buffer, curr));
```

Comment:
You want to use `outputs` and the context for json_strdup here: it's a nice convention that a container owns all its members.

Reply:
In `listfunds` we use `txid` and `output` fields.  Since we have JSON anyway, I suggest sticking with that, rather than using a boutique encoding for outpoints?

Reply:
> In listfunds we use txid and output fields. Since we have JSON anyway, I suggest sticking with that, rather than using a boutique encoding for outpoints?

Does it mean that the user should pass an array of objects ? `[{"txid":"....", "output":0}, ...]`